index.html

```diff
@@ -0,0 +1,248 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Dunk card display</title>
+		<link rel="stylesheet" type="text/css" href="quize.css">
+	</head>
+	<body>
+
+		<article>
+			<header>
+				<h1>2017 Nissan 370Z Roadster</h1>
+			</header>
+
+			<section class="section1">
+
+				<img src="http://lln.mnmcdn.com/photos/cars/nissan/big-images/2013-nissan-370z-roadster-2_653.jpg">
```

Comment:
So this totally works and is fine. But technically the requirement asks that 'The first section should contain three child block elements.' and 5-i asks 'The first element contains the product image.'
It would be great practice to put your images in a block level element (such as a div) and then try to recenter them.